reference/system-databases/cluster-load.md

```diff
@@ -0,0 +1,65 @@
+---
+title: CLUSTER_LOAD
+summary: Learn the `CLUSTER_LOAD` cluster load table.
+category: reference
+---
+
+# CLUSTER_LOAD
+
+The `CLUSTER_LOAD` cluster load table provides the current load information of the server where each instance of the TiDB cluster is located.
+
+{{< copyable "sql" >}}
+
+```sql
+desc cluster_load;
+```
+
+```
```

Comment:
Adding `sql` to the results can also make it looks better with colors at the website.
```suggestion
```sql
```